app/views/shared/_navbar.html.erb

```diff
@@ -24,12 +24,12 @@
           <%= image_tag "https://kitt.lewagon.com/placeholder/users/ssaunier", class: "avatar dropdown-toggle", id: "navbarDropdown", data: { toggle: "dropdown" }, 'aria-haspopup': true, 'aria-expanded': false %>
           <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown">
             <% if current_user.kind == "donator" %>
-              <%= link_to "Your products", "#", class: "dropdown-item" %>
-              <%= link_to "Bookings", "#", class: "dropdown-item" %>
+              <%= link_to "Donate", new_product_path, class: "dropdown-item" %>
+              <%= link_to "Your donations", products_path, class: "dropdown-item" %>
```

Comment:
For the 'Your Donations' link, I think we will want it to go a "my bookings" page, no? products_path is the index of all the products. We don't have a my bookings page currently however, or have planned it much.

What do we do about this?